std_msgs/Header header
NamedPose[] poses
